jobs-builder/jobs/pr.yaml

```diff
@@ -0,0 +1,180 @@
+          url: https://github.com/kata-containers/kata-containers
+          branches:
+            - '*/{branch}'
+          wipe-workspace: false
+    triggers:
+      - timed: 'H 0 * * *'
+    builders:
+      - shell:
+          !include-raw: include/ci_entrypoint.sh.inc
+    <<: *main_jobs_common_publishers
+
+- job-template:
+    # Use to generate pull request (PR) jobs.
+    #
+    # Arguments:
+    #   repo        - the repository name.
+    #   os          - the node Operating System in <name>-<version> format.
+    #   arch        - the node architecture (e.g x86_64, s390x, ppc64le, and so on).
+    #   ci_job      - the CI job type as defined in https://github.com/kata-containers/tests/blob/main/.ci/ci_job_flags.sh
+    #
+    name: "{repo}-{os}-{arch}-{ci_job}-PR"
+    <<: *main_jobs_common_properties
+    # Allow concurrent jobs by default. Specify `false` on the project definition otherwise.
+    concurrent_toggle: true
+    description:
+      !j2: |
+         <pre>
+         Pull Request (PR) job.
+         OS="{{ os }}"
+         arch="{{ arch }}"
+         CI_JOB="{{ ci_job }}"
+         repo="{{ repo }}"
+         type="PR"
+         </pre>
+    scm:
+      - git:
+          url: https://github.com/kata-containers/{repo}
+          branches:
+            - '${{sha1}}'
+          refspec: '+refs/pull/${{ghprbPullId}}/*:refs/remotes/origin/pr/${{ghprbPullId}}/*'
+          wipe-workspace: false
+    properties:
+      - github:
+          url: https://github.com/kata-containers/{repo}
+    triggers:
+      - github-pull-request:
+          auth-id: 'katacontainers'
+          github-hooks: true
+          # Trigger only on commenting phrase in the pull request.
+          only-trigger-phrase: true
+          # The expected phrase will be like "/(re)test-<OS Name>"
+          trigger-phrase:
+            !j2: '.*(\n|^|\s)/(re)?test(-{{ os.split("-")[0] }})?(\n|$|\s)+.*'
+          # Skip on commenting phrase.
+          skip-build-phrase: '.*\[skip\W+ci\].*'
+          cron: 'H/5 * * * *'
+          # List of organizations whose members are allowed to build.
+          org-list:
+            - kata-containers
+          # Members of allowed organizations will have admin rights.
+          allow-whitelist-orgs-as-admins: true
+          # Branches allowed to be tested.
+          white-list-target-branches:
+            - main
+            - stable-2.*
+          # Branches disallowed to be tested.
+          black-list-target-branches:
+            - master
+            - stable-1.*
+    builders:
+      - shell:
+          !include-raw: include/ci_entrypoint.sh.inc
+    <<: *main_jobs_common_publishers
+
+###
+# Define the projects
+###
+- project:
+    name: "Generate jobs for CRIO_K8S"
+    repo:
+      - kata-containers
+      - tests
+    branch:
+      - main
+    os:
+      - fedora-35
+    arch:
+      - x86_64
+    ci_job:
+      - CRIO_K8S
+      - CRIO_K8S_MINIMAL
+    maintainers:
+      - '@snir911'
+      - '@c3d'
```

Comment:
ooi, are you both in the same TZ? If so, maybe we could add a few more users: I'd say minimum of three, but even better would be to add a single GH team here.

Reply:
We agreed that baseline jobs should be owners (https://github.com/kata-containers/ci/issues/403). @snir911 and @c3d are going to be the owners of those Fedora 35  jobs. So it is fine.

Reply:
What does having a user specified in `maintainers` here actually mean though? What if they are both unavailable? Is it simply for mail notification or similar?

Reply:
I'm not sure I understood your first question. The `maintainers` variable is used on the description of the job, so on the Jenkins UI it will be listed the names on the jobs' description. An example here: http://jenkins.katacontainers.io/view/Daily%20baseline/job/kata-containers-main-fedora-35-x86_64-CRIO_K8S-baseline/

The idea is to have their names on the job's description for the case a developer need to speak with the owners (e.g. developer's PR cannot be merged because a job is failing and she/he notices the baseline has failed too). The owners are also listed in a table on the view [here](http://jenkins.katacontainers.io/view/Daily%20baseline/). Having their names on the job's description just increase visibility. Do you see any problem with that?